css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+body {
+  font-family: Lato, sans-serif;
+}
+
+.container {
+  max-width: 900px;
+  margin: auto;
+}
+
+.blue-backg {
+  background-color: #1996da;
+}
+
+.white-text {
+  color: white;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+}
+
+.icons {
+  width: 100%;
+}
+
+.white-circle {
+  margin: auto;
+  border-radius: 100%;
+  background-color: white;
+  width: 40%;
+  padding: 1.5rem;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.letter-space {
+  letter-spacing: 0.1rem;
+}
+
+.description {
+  line-height: 1rem;
+  font-size: 12px;
+  opacity: 0.7;
```

Comment:
Oh no, you shouldn't use `opacity` to adjust color, you should just use the `color` property